### DISCUSSION TITLE
cron with API

### DISCUSSION THREAD

**Discussion by TWART016** [firefly-iii/firefly-iii#5113] [Firefly III] [answered]
Hi,

I want to start the cron with this URL:
http://192.168.178.15:10000/api/v1/cron/1ANhBUPZkeeGtDuB05cYBhJcHImPmk5mnDkE2Wwt

I get the error:  404 - Page not found

The token I generated in OAuth Clients.

version 5.6.0

**Comment by JC5** [comment, marked as answer]
You need the token from the /profile page.

**Reply by TWART016** [reply]
Thanks. It is in the tab: command line token.
